modules/manage/partials/schema-compatibility.adoc

```diff
@@ -52,6 +52,9 @@ Relax constraints, for example:
 * Decrease `minItems`, `minLength`, or `minProperties`; increase `maxItems`, `maxLength`, `maxProperties`
 * Add more property types (for example, `"type": "integer"` to `"type": ["integer", "string"]`)
 * Add more enum values
+* Reduce `multipleOf` by an integral factor
+* Relaxing additional properties if `additionalProperties` was not previously specified as `false`
+* Removing a `unqiueItems` that was `false`
```

Comment:
```suggestion
* Removing a `uniqueItems` that was `false`
```

Reply:
Wow. I had to stare at that an embarrassingly long to see the difference.

Reply:
https://github.com/redpanda-data/docs/pull/666